MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -1604,6 +1604,9 @@ public static int calculateEffectiveBV(AtBDynamicScenario scenario, Campaign cam
                 bvBudget += forceBVBudget;
             }
         }
+        
+        bvBudget += bvBudget * scenario.getEffectivePlayerBVMultiplier();
```

Comment:
Needs a Math.round, Math.floor, or Math.ceil

Reply:
Why? It always produces a whole number.

Reply:
It is also an unrounded double to integer cast, and thus may not in future... and causes another CE issue.

Reply:
Not really sure why we care if the bv budget drifts one way or the other by 1 point - we're usually talking thousands or tens of thousands of point.

Reply:
And honestly, how often does java change the behavior of their "double to integer" cast? It's not like we're directly executing CPU instructions here.

Reply:
Rarely, and I know it is minor... but it's still a code scanning alert both in LGTM and in GitHub Actions, and I'm trying to keep the first low while reducing the latter.